docs/subquery_network/node_operators/setup/becoming-a-node-operator.md

```diff
@@ -204,16 +204,109 @@ Once you have successfully logged in, look for 'dashboards' on the left-hand sid
 ![grafana_query_count](/assets/img/network/grafana_query_count.png)
 ![grafana_query_stats](/assets/img/network/grafana_query_stats.png)
 
-## 10. Upgrade Node Operator services (Ongoing)
+## 10. Auto-Upgrading Docker Compose
 
-To upgrade a Node Operator service, you will need to update the version of the image used in the docker-compose file. This can be done by updating the image field in the service definition to the new version you want to use.
+### Step 1: Download the `auto-upgrade-tool.sh` Script
 
-Once the image version has been updated in the docker-compose file, you can restart the specific container that needs to be upgraded. This can be done by running the following command in the terminal:
+Use the following command to download the `auto-upgrade-tool.sh` script:
 
-```bash
-docker-compose up -d --no-deps container-service-name
+```sh
+curl https://raw.githubusercontent.com/subquery/network-indexer-services/main/deploy/auto-upgrade-tool.sh -o auto-upgrade-tool.sh
+```
+
+### Step 2: Usage of `auto-upgrade-tool.sh`
+
+The `auto-upgrade-tool.sh` script automatically fetches the latest tags for `subquerynetwork/indexer-coordinator` and `subquerynetwork/indexer-proxy` from Docker Hub and updates the versions in your Docker Compose configuration file.
+
+```sh
+# Default usage with `docker-compose.yml`
+./auto-upgrade-tool.sh
```

Comment:
is `chmod +x ./auto-upgrade-tool.sh` required?

Reply:
also, user should not run this in a one-off session, otherwise it will exit once the session is closed.

Reply:
> is `chmod +x ./auto-upgrade-tool.sh` required? 是必需的吗?

add chmod doc

Reply:
> also, user should not run this in a one-off session, otherwise it will exit once the session is closed.

not needed, docker-compose will run in the background